services/skus/receipt.go

```diff
@@ -83,17 +54,36 @@ func (dt *dumpTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	return resp, rtErr
 }
 
+type androidPublisher struct {
```

Comment:
suggestion: It does look like a `playStoreClient` to me. `androidPublisher` does not fully describe what it does (we certainly don't publish anything).

Reply:
This is simply just a wrapper around Google `androidpublisher` api which is their naming convention, the intention is to simply wrap it and not imply there is any extra functionality etc. IMO when looking up docs it is also a little clear if it simply wrap their naming.